### ISSUE TITLE
enableAnalogSticks not working

### ISSUE THREAD

**Issue by rvera** [SukkoPera/PsxNewLib#21]
Hi, I'm running this example

https://github.com/SukkoPera/PsxNewLib/blob/9b88bdda232b83a1b736be81c6973c0629072ede/examples/PSX2USB/PSX2USB.ino

and enableAnalogSticks fails for me on an official ps2 controller. Any idea of what could be going on?
- Running on a pro micro
- Using 3.3v 500ma as power for the controller as I read online that feeding it the full 5v 1ma could hurt my controllers

**Comment by SukkoPera** [comment]
Are you using 1k pull-ups on CMD and DAT?

**Comment by rvera** [comment]
you mean if those pins are 5v 1A? Yes, only on the power pin I used some resistances to step down to 3.3v

**Comment by SukkoPera** [comment]
No, this reminds me again I need to add proper instructions for connecting the controller to the Arduino.

**Comment by rvera** [comment]
That would be awesome, would love to see some official documentation about this

**Comment by SukkoPera** [comment]
Until I do that, you can follow [the schematics of my shield(https://github.com/SukkoPera/PsxControllerShield/blob/master/doc/schematics.pdf).
